registration/tests/test_views.py

```diff
@@ -532,7 +527,7 @@ def test_modifying_design_fields_sends_email(self):
         # Modifying most fields shouldn't send an e-mail
         with self.assertTemplateNotUsed('registration/fields_changed_email.txt'):
             modified.update({
-                'project_description': 'Learn',
+                'status': 'Pending',
```

Comment:
This test simulates that the user manually modified some of the fields by typing (when returning to the form after having submitted it). „status“ isn't one of those visible fields, it's something internal. So better replace `project_description` by a visible field from https://github.com/open-craft/opencraft/blob/adam%2Fregistration-updates/registration/forms.py#L112, I'd suggest `full_name` because it's also text, but you can also use the checkbox `subscribe_to_updates` or maybe another.